src/OrchardCore.Modules/OrchardCore.Facebook/AdminMenu.cs

```diff
@@ -1,72 +1,108 @@
-                            .Action("Index", "Admin", new { area = "OrchardCore.Settings", groupId = FacebookConstants.Features.Core })
-                            .Permission(Permissions.ManageFacebookApp)
-                            .LocalNav())
-                        )
-                    );
-            }
             return Task.CompletedTask;
         }
+
+        builder
+            .Add(S["Security"], security => security
+                .Add(S["Authentication"], authentication => authentication
+                    .Add(S["Facebook"], S["Facebook"].PrefixPosition(), settings => settings
+                        .AddClass("facebook")
+                        .Id("facebook")
+                        .Action("Index", "Admin", new { area = "OrchardCore.Settings", groupId = FacebookConstants.Features.Login })
+                        .Permission(Permissions.ManageFacebookApp)
+                        .LocalNav()
+                    )
+                )
+            );
+
+        return Task.CompletedTask;
     }
+}
+
+public class AdminMenuPixel : INavigationProvider
+{
+    private readonly IStringLocalizer S;
 
-    [Feature(FacebookConstants.Features.Login)]
-    public class AdminMenuLogin : INavigationProvider
+    public AdminMenuPixel(
+        IStringLocalizer<AdminMenuLogin> stringLocalizer)
     {
-        private readonly ShellDescriptor _shellDescriptor;
-        private readonly IStringLocalizer S;
+        S = stringLocalizer;
+    }
 
-        public AdminMenuLogin(
-            IStringLocalizer<AdminMenuLogin> localizer,
-            ShellDescriptor shellDescriptor)
+    public Task BuildNavigationAsync(string name, NavigationBuilder builder)
+    {
+        if (!String.Equals(name, "admin", StringComparison.OrdinalIgnoreCase))
         {
-            S = localizer;
-            _shellDescriptor = shellDescriptor;
+            return Task.CompletedTask;
         }
 
-        public Task BuildNavigationAsync(string name, NavigationBuilder builder)
-        {
-            if (String.Equals(name, "admin", StringComparison.OrdinalIgnoreCase))
-            {
-                builder.Add(S["Security"], security => security
-                        .Add(S["Authentication"], authentication => authentication
-                        .Add(S["Facebook"], S["Facebook"].PrefixPosition(), settings => settings
-                        .AddClass("facebook").Id("facebook")
-                            .Action("Index", "Admin", new { area = "OrchardCore.Settings", groupId = FacebookConstants.Features.Login })
-                            .Permission(Permissions.ManageFacebookApp)
-                            .LocalNav())
-                ));
-            }
+        builder
+            .Add(S["Configuration"], configuration => configuration
+                .Add(S["Settings"], settings => settings
+                    .Add(S["Facebook Pixel"], S["Facebook Pixel"].PrefixPosition(), pixel => pixel
```

Comment:
An icon like for the other two features would be nice:
![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/0be364b4-95b5-4357-8e81-14f763845db1)

Reply:
added! All icons are now showing the Meta icon instead of the Facebook icon.